pkg/apis/deployment/v1/conditions_params.go

```diff
@@ -0,0 +1,44 @@
+//
+// DISCLAIMER
+//
+// Copyright 2016-2022 ArangoDB GmbH, Cologne, Germany
```

Comment:
not sure about copyright notice in operator, however in Oasis it's 
`Copyright 2022` for new files and `Copyright x-2022`
 for updates